include/pmacc/mappings/kernel/MappingDescription.hpp

```diff
@@ -87,6 +87,11 @@ namespace pmacc
             return this->gridSuperCells;
         }
 
+        HDINLINE DataSpace<DIM> getGridSuperCellsWithoutGuards() const
+        {
+            return this->gridSuperCells - 2 * this->guardingSuperCells;
```

Comment:
QUESTION: Does the PMAcc model guarantee that the guards are symmetric?

Reply:
I actually not sure, @psychocoderHPC care to chime in?

Reply:
Guarding supercells can be different per dimension.

Why is this change here in this PR? looks like it is already merged.

Reply:
@psychocoderHPC But are they the same for the two different directions of one dimension? 

I would assume yes, since MappingDescription stores only one value per dimension.

Note: This method is required by in the FLYonPIC SuperCell fields which are newly added with this PR, therefore this change also belongs to this PR

Reply:
Yes guard size is per dimension and equal for the negative and positive direction.